gitbutler-ui/src/lib/menuBarController.ts

```diff
@@ -0,0 +1,43 @@
+import { invoke, listen as listenIpc } from '$lib/backend/ipc';
+import { unsubscribe } from '$lib/utils/unsubscribe';
+import { goto } from '$app/navigation';
+
+export class MenuBarController {
```

Comment:
The concepts of controller and service are in my head a bit interchangeable. Do you have any thoughts on what we should prefer, and how to decide if both?